hieradata/common.yaml

```diff
@@ -52,10 +52,17 @@ psick::base::linux_classes:
 
 # PSICK classification for Windows nodes
 psick::pre::windows_classes:
+  chocolatey: chocolatey
   hosts: psick::hosts::resource
 psick::base::windows_classes:
+  packages: psick::packages
   time: '::psick::time'
 
+psick::packages::packages_list:
```

Comment:
@antgel be careful here, if linux servers include the psick::packages class they will try to install these packages as well
the class psick::packages currently has no os specific params